client/my-sites/earn/memberships/index.jsx

```diff
@@ -201,19 +210,78 @@ class MembershipsSection extends Component {
 		);
 	}
 
-	render() {
+	renderStripeConnected() {
 		return (
 			<div>
 				{ this.renderEarnings() }
 				{ this.renderSubscriberList() }
 			</div>
 		);
 	}
+
+	renderConnectStripe() {
+		return (
+			<div>
+				<SectionHeader label={ this.props.translate( 'Stripe Connection' ) } />
+				<Card>
+					<div className="memberships__module-content module-content">
+						<p>
+							{ this.props.translate(
+								'Start collecting subscription payments! Recurring payments are processed through Stripe. Click the button below to create a new account or to connect existing Stripe account.'
+							) }
+						</p>
+						<StripeConnectButton href={ this.props.connectUrl } target="_blank">
+							{ this.props.translate( 'Connect with Stripe' ) }
+						</StripeConnectButton>
+					</div>
+				</Card>
+			</div>
+		);
+	}
+
+	render() {
+		if ( this.props.isJetpackTooOld ) {
+			return (
+				<Notice
+					status="is-warning"
+					text={ this.props.translate(
+						'Please update Jetpack plugin to version 7.3 or higher in order to use the Membership button block'
+					) }
+					showDismiss={ false }
+				>
+					<NoticeAction
+						href={ `https://wordpress.com/plugins/jetpack/${ this.props.siteSlug }` }
+						icon="external"
+					/>
+				</Notice>
+			);
+		}
+
+		if ( ! this.props.paidPlan ) {
+			return (
+				<UpgradeNudge
+					plan={ this.props.isJetpack ? PLAN_JETPACK_PERSONAL : PLAN_PERSONAL }
+					shouldDisplay={ () => true }
+					feature={ FEATURE_MEMBERSHIPS }
+					title={ this.props.translate( 'Upgrade to the personal plan' ) }
```

Comment:
```suggestion
					title={ this.props.translate( 'Upgrade to the Personal plan' ) }
```

(I think it'd be better if the Upgrade Nudge Expanded component is used here instead tho to be honest, but the capital letter is probably worth updating regardless)